index.js

```diff
@@ -182,6 +182,8 @@ EasyXml.prototype.parseChildElement = function(parentXmlNode, parentObjectNode)
                     } else {
                         parentXmlNode.set(key.substring(1), child);
                     }
+                } else if (typeof child === 'object' && typeof child.toString === 'function') {
```

Comment:
Agreed, I think this feature needs a little more thinking, it was a quick way to make it work, I'll study a more robust solution and add a test for it.